events/models/profiles.py

```diff
@@ -705,7 +713,11 @@ def save(self, *args, **kwargs):
 
 class Topic(models.Model):
     category = models.ForeignKey(
-        Category, on_delete=models.CASCADE, null=False, blank=False
+        Category,
+        verbose_name = "Category"
```

Comment:
There's a missing a comma here  :) 
```suggestion
        verbose_name = "Category",
```